docs_src/OVMS/pipelinesetup.md

```diff
@@ -1,96 +1,96 @@
 
-```bash
-git clone  https://github.com/intel-retail/automated-self-checkout.git && cd ./automated-self-checkout
-```
-
-### Step 2: Build profile launcher
-
-To build the profile-launcher binary executable, just use `make build-profile-launcher` from the command line.
-
-Each profile is an unique pipeline use case.  We provide some profile examples, and the configuration examples of profiles [are located here](https://github.com/intel-retail/automated-self-checkout/tree/main/configs/opencv-ovms/cmd_client/res).  [Go here](profileLauncherConfigs.md) to find out the detail descriptions for the configuration of profile used by profile launcher.
-
-!!! Note Optional to install Golang 1.20
-
-For the development debugging purpose like running and debugging in IDE, you may want to install Golang 1.20. Here is [the link to download](https://go.dev/dl/) Golang.
-
-Follow the [installation instruction](https://go.dev/doc/install#Go_installation) with the same downloaded file name above for version 1.20.
-
-### Step 3: Build the benchmark Docker images
-
-For use cases that support benchmarking, build the benchmark Docker images:
-
-```bash
-cd benchmark-scripts
-make build-all
-```
-
-### Step 4: Download the models manually (Optional)
-
-When `run.sh` the Model downloader script is automatically called. the model downloader script is automatically called as part of run-ovms.sh (part of run.sh). You can also download the models manually using the model downloader script:
-
-```bash
-sh ./download_models/getModels.sh --workload ovms
-```
-
-!!! Note
-    Depending on your internet connection, this might take less than a minute.
-
-
-### Step 5: Download image file Manually (Optional)
-
-The sample image downloader script is automatically called as part of run-ovms.sh. You can also download the sample image manually using script below:
-
-```bash
-sh ./configs/opencv-ovms/scripts/image_download.sh 
-```
-
-!!! Note
-    Depending on your internet connection, this might take less than a minute.
+    ```bash
+    git clone  https://github.com/intel-retail/automated-self-checkout.git && cd ./automated-self-checkout
+    ```
 
+2. Build the profile launcher binary executable
 
-### Step 6: Download bit model Manually (optional)
+    ```bash
+    make build-profile-launcher
+    ```
 
-Here is the script to build container for bit model downloading:
+    Each profile is an unique pipeline use case.  We provide some profile examples, and the configuration examples of profiles [are located here](https://github.com/intel-retail/automated-self-checkout/tree/main/configs/opencv-ovms/cmd_client/res).  [Go here](profileLauncherConfigs.md) to find out the detail descriptions for the configuration of profile used by profile launcher.
 
-```bash
-docker build -f Dockerfile.bitModel -t bit_model_downloader:dev .
-```
+    !!! Note 
+        Optional to install Golang 1.20
 
-Here is the script to run container and downloads the bit models:
+    !!! Debug
```

Comment:
We can just leave this part and remove the note above. Golang only for development/debug purposes.